### DISCUSSION TITLE
Failed to load plugin turbo

### DISCUSSION THREAD

**Discussion by ilhamfu** [vercel/turborepo#3755] [Help] [answered]
for some reason, the shared eslint got broken. It says  that it cannot find eslint-plugin-turbo

**Comment by anthonyshew** [comment, marked as answer]
Looking at the date here, I'm going to close this as stale. There have been many patches since then!

If you're still running into this problem, check out the `examples` directory for some guidance or open an issue if you believe there is a bug.
